Applications/Spire/Include/Spire/StyleParser/DataTypes/FloatType.hpp

```diff
@@ -0,0 +1,19 @@
+#ifndef SPIRE_FLOAT_TYPE_HPP
+#define SPIRE_FLOAT_TYPE_HPP
+#include "Spire/StyleParser/DataTypes/DataType.hpp"
+
+namespace Spire {
+
+  /** Represents the floating-point type. */
+  class FloatType : public DataType {
```

Comment:
Can this be further inherited from?

Reply:
I think the percent type might inherit from it.

Reply:
Avoid mixing types and values. A percentage is nothing more than a number divided by 100.